### PR TITLE
Update new completions for Flutter

### PR DESCRIPTION
**Pull request by kamleshkc2002**
closes https://github.com/zsh-users/zsh-completions/issues/682

Add new completions for the Flutter CLI

Updated as per the original issue.

- [X] This compdef is not already available in zsh.
- [ ] This compdef is not already available in their original project.
- [ ] I am the original author, or I have authorization to submit this work.
- [X] This is a finished work.
- [ ] It has a header containing authors, status and origin of the script.
- [X] It has a license header or I accept that it will be licensed under the terms of the Zsh license.

### CODE REVIEW

File: src/_flutter [hunk 1]
```diff
@@ -380,7 +404,12 @@ _root_commands() {
     'test:Run Flutter unit tests for the current project.'
     'trace:Start and stop tracing for a running Flutter app.'
     'update-packages:Update the packages inside the Flutter repo.'
-    'upgrade:Upgrade your copy of Flutter.')
+    'upgrade:Upgrade your copy of Flutter.'
+    'assemble:Assemble and build flutter resources.'
```
Comment (syohex) [inline src/_flutter:408]: Original code sorts sub-command by alphabetical order. Could you insert new commands by alphabetical order and keep alphabetical order ?

File: src/_flutter [hunk 2]
```diff
@@ -340,7 +340,31 @@ _arguments -C -A "-*" \
             '(-h --help)'{-h,--help}'[Print this usage information.]' \
           && ret=0
         ;;
-        
+        (assemble)
```
Comment (syohex) [inline src/_flutter:343]: Same as other comment.

Comment (kamleshkc2002) [reply src/_flutter]: sounds good.. updated.

File: src/_flutter (hunk 1 above)
Comment (kamleshkc2002) [reply src/_flutter]: sounds good. updated.

**Comment by syohex** [comment]
Thanks for updating code.

**Pull request merged**